deploy/template.yaml

```diff
@@ -168,6 +178,8 @@ Mappings:
       nodeOldSpaceLimit: "1792"
       nodeEnv: "production"
       desiredTaskCount: 2
+      ga4Disabled: true
+      uaDisabled: true
```

Comment:
Based on https://govukverify.atlassian.net/browse/DCP-3156?focusedCommentId=126209 this should initially be true, false in prod (and integration)?